  practice9.2.1.c
d:\new_project\chapter9\practice9.2.1\practice9.2.1\practice9.2.1.c(18): warning C4013: 関数 'exist' は定義されていません。int 型の値を返す外部関数と見なします。
d:\new_project\chapter9\practice9.2.1\practice9.2.1\practice9.2.1.c(21): warning C4047: '==': 間接参照のレベルが 'errno_t' と 'void *' で異なっています。
d:\new_project\chapter9\practice9.2.1\practice9.2.1\practice9.2.1.c(26): warning C4047: '関数': 間接参照のレベルが 'FILE *' と 'FILE **' で異なっています。
d:\new_project\chapter9\practice9.2.1\practice9.2.1\practice9.2.1.c(26): warning C4024: 'fgetc': の型が 1 の仮引数および実引数と異なります。
d:\new_project\chapter9\practice9.2.1\practice9.2.1\practice9.2.1.c(29): warning C4047: '関数': 間接参照のレベルが 'FILE *' と 'FILE **' で異なっています。
d:\new_project\chapter9\practice9.2.1\practice9.2.1\practice9.2.1.c(29): warning C4024: 'fclose': の型が 1 の仮引数および実引数と異なります。
practice9.2.1.obj : error LNK2019: 未解決の外部シンボル _exist が関数 _main で参照されました。
D:\new_project\chapter9\practice9.2.1\Debug\practice9.2.1.exe : fatal error LNK1120: 1 件の未解決の外部参照
